### PR TITLE
Bump requirements to be in line with SLE12 SP1

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -19,9 +19,9 @@ source "https://rubygems.org"
 
 group :development do
   gem "closure-compiler", "~> 1.1.10"
-  gem "sass", "~> 3.2.19"
+  gem "sass", "~> 3.4.13"
   gem "sprockets-standalone", "~> 1.2.1"
-  gem "sprockets", "~> 2.11.0"
+  gem "sprockets", "~> 3.1.0"
```
Comment (tboerger) [inline Gemfile]: As I can see you have not checked if it still works. The dependencies can't get resolved.
Reference: https://rubygems.org/gems/sprockets-standalone

Comment (rsalevsky) [inline Gemfile]: I think we should drop `sprockets-standalone` completely.

Comment (dirkmueller) [inline Gemfile]: right, standalone requires 2.11. I'm happy if we could drop that

**Comment by toabctl** [comment]
+1

File: crowbar_framework/config/boot.rb [hunk 2]
```diff
@@ -55,7 +55,7 @@
   gem "dotenv", version: "~> 1.0"
   require "dotenv"
 
-  gem "hashie", version: "~> 2.1"
+  gem "hashie", version: "~> 3.4"
```
Comment (aplanas) [inline crowbar_framework/config/boot.rb]: In another PR you suggested to use patch level versioning. Still makes sense here?

**Comment by rsalevsky** [comment]
+1

**Comment by vuntz** [comment]
What about js-routes?

File: crowbar_framework/config/boot.rb (hunk 2 above)
Comment (dirkmueller) [inline crowbar_framework/config/boot.rb]: yeah, I still think, but this patch is more urgent than the other one. I expect the other one to be discussed for a couple of months..

**Comment by dirkmueller** [comment]
@vuntz fixed

**Comment by aplanas** [comment]
+1

**Comment by vuntz** [comment]
+1

File: Gemfile (hunk 1 above)
Comment (tboerger) [inline Gemfile]: This can get dropped on crowbar-core, but not from any other component. So we will still have this issue.

**Pull request merged**